<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="4" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="72910" started-at="2020-06-16T19:16:15 IST" finished-at="2020-06-16T19:17:28 IST">
    <groups>
      <group name="login">
        <method signature="NewTest.Tc1()[pri:1, instance:p1.NewTest@3108bc]" name="Tc1" class="p1.NewTest"/>
        <method signature="NewTest.Tc2()[pri:2, instance:p1.NewTest@3108bc]" name="Tc2" class="p1.NewTest"/>
        <method signature="NewTest.Tc3()[pri:3, instance:p1.NewTest@3108bc]" name="Tc3" class="p1.NewTest"/>
        <method signature="NewTest.Tc4()[pri:4, instance:p1.NewTest@3108bc]" name="Tc4" class="p1.NewTest"/>
      </group> <!-- login -->
    </groups>
    <test name="Default test" duration-ms="72910" started-at="2020-06-16T19:16:15 IST" finished-at="2020-06-16T19:17:28 IST">
      <class name="p1.NewTest">
        <test-method status="PASS" signature="be()[pri:0, instance:p1.NewTest@3108bc]" name="be" is-config="true" duration-ms="10888" started-at="2020-06-16T19:16:15 IST" finished-at="2020-06-16T19:16:26 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- be -->
        <test-method status="FAIL" signature="Tc1()[pri:1, instance:p1.NewTest@3108bc]" name="Tc1" duration-ms="3455" started-at="2020-06-16T19:16:26 IST" finished-at="2020-06-16T19:16:29 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Minu vaade - MantisBT] but found [My View - MantisBT]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Minu vaade - MantisBT] but found [My View - MantisBT]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at p1.NewTest.Tc1(NewTest.java:20)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc1 -->
        <test-method status="PASS" signature="am()[pri:0, instance:p1.NewTest@3108bc]" name="am" is-config="true" duration-ms="4808" started-at="2020-06-16T19:16:29 IST" finished-at="2020-06-16T19:16:34 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method status="PASS" signature="be()[pri:0, instance:p1.NewTest@3108bc]" name="be" is-config="true" duration-ms="10305" started-at="2020-06-16T19:16:34 IST" finished-at="2020-06-16T19:16:44 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- be -->
        <test-method status="PASS" signature="Tc2()[pri:2, instance:p1.NewTest@3108bc]" name="Tc2" duration-ms="1489" started-at="2020-06-16T19:16:44 IST" finished-at="2020-06-16T19:16:46 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc2 -->
        <test-method status="PASS" signature="am()[pri:0, instance:p1.NewTest@3108bc]" name="am" is-config="true" duration-ms="5192" started-at="2020-06-16T19:16:46 IST" finished-at="2020-06-16T19:16:51 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method status="PASS" signature="be()[pri:0, instance:p1.NewTest@3108bc]" name="be" is-config="true" duration-ms="10446" started-at="2020-06-16T19:16:51 IST" finished-at="2020-06-16T19:17:02 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- be -->
        <test-method status="PASS" signature="Tc3()[pri:3, instance:p1.NewTest@3108bc]" name="Tc3" duration-ms="2765" started-at="2020-06-16T19:17:02 IST" finished-at="2020-06-16T19:17:04 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc3 -->
        <test-method status="PASS" signature="am()[pri:0, instance:p1.NewTest@3108bc]" name="am" is-config="true" duration-ms="5138" started-at="2020-06-16T19:17:04 IST" finished-at="2020-06-16T19:17:09 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
        <test-method status="PASS" signature="be()[pri:0, instance:p1.NewTest@3108bc]" name="be" is-config="true" duration-ms="11508" started-at="2020-06-16T19:17:09 IST" finished-at="2020-06-16T19:17:21 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- be -->
        <test-method status="FAIL" signature="Tc4()[pri:4, instance:p1.NewTest@3108bc]" name="Tc4" duration-ms="1458" started-at="2020-06-16T19:17:21 IST" finished-at="2020-06-16T19:17:22 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Minu vaade - MantisBT] but found [MantisBT]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Minu vaade - MantisBT] but found [MantisBT]
at org.testng.Assert.fail(Assert.java:97)
at org.testng.Assert.assertEqualsImpl(Assert.java:136)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:575)
at org.testng.Assert.assertEquals(Assert.java:585)
at p1.NewTest.Tc4(NewTest.java:46)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:584)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:172)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:804)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:145)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(Unknown Source)
at org.testng.TestRunner.privateRun(TestRunner.java:770)
at org.testng.TestRunner.run(TestRunner.java:591)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:402)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:396)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:355)
at org.testng.SuiteRunner.run(SuiteRunner.java:304)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1180)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1102)
at org.testng.TestNG.runSuites(TestNG.java:1032)
at org.testng.TestNG.run(TestNG.java:1000)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Tc4 -->
        <test-method status="PASS" signature="am()[pri:0, instance:p1.NewTest@3108bc]" name="am" is-config="true" duration-ms="5383" started-at="2020-06-16T19:17:22 IST" finished-at="2020-06-16T19:17:28 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- am -->
      </class> <!-- p1.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
